modular_citadel/code/game/objects/items/defib.dm

```diff
@@ -0,0 +1,42 @@
+/obj/item/defibrillator
+	var/healdisk = FALSE // Will we shock people dragging the body?
+	var/pullshocksafely = FALSE //Dose the unit have the healdisk upgrade?
+	var/fastasfuck = 0 // is the defib faster
+	var/snowflake = 10
+
+/obj/item/twohanded/shockpaddles/shock_touching(dmg, mob/H)
+	if(defib.pullshocksafely && isliving(H.pulledby))
+		H.visible_message("<span class='danger'>The defibrillator safely discharges the excessive charge into the floor!</span>")
```

Comment:
This is insane, the grounding rod would need to be attached at the least point somewhere or you'd end up doing nothing at all or shocking yourself and the unit.